AFNetworking/AFURLRequestSerialization.m

```diff
@@ -578,8 +578,15 @@ - (void)encodeWithCoder:(NSCoder *)coder {
 
 - (instancetype)copyWithZone:(NSZone *)zone {
     AFHTTPRequestSerializer *serializer = [[[self class] allocWithZone:zone] init];
-    dispatch_sync(self.requestHeaderModificationQueue, ^{
-        serializer.mutableHTTPRequestHeaders = [self.mutableHTTPRequestHeaders mutableCopyWithZone:zone];
+
+    // \c HTTPRequestHeaders is accessed and not \c mutableHTTPRequestHeaders, despite of the extra
+    // copy and context-switch overhead, in order to promise thread-safe access to the mutable
```

Comment:
`thread-safe access to the headers dictionary`?
Since we don't access the mutable dictionary.

Reply:
Done.